examples/scripts/ExampleMultipleChaperones.py

```diff
@@ -54,6 +54,7 @@
 plt.subplot(131)
 plt.imshow(det.assemble_image_stack(patternOp),norm=LogNorm())
 plt.colorbar()
+<<<<<<< HEAD
```

Comment:
looks like these are not merged properly. is that true?

Reply:
The merge seems to have gone wrong somehow..pushed again.